song-go-client/song/endpoint.go

```diff
@@ -58,15 +49,6 @@ func truth(condition bool) string {
 	return "false"
 }
 
-// Save saves the specified uploadID as an analysis assuming it had passed validation
-func (s *Endpoint) Save(studyID string, uploadID string, ignoreCollisions bool) url.URL {
```

Comment:
you mentioned `upload` being replaced... also `save`?

Reply:
Right. The SONG `submit` endpoint merges the `upload` and `save`.

Reply:
hmm I remember that whole epic now. cool!